lib/main_common.pm

```diff
@@ -1099,6 +1099,7 @@ sub load_consoletests {
     loadtest "console/system_state";
     loadtest "console/prepare_test_data";
     loadtest "console/consoletest_setup";
+    loadtest 'console/systemd_wo_udev' unless (is_sle('<15-sp4') || is_leap('<15.4'));
```

Comment:
Maybe nitpicking, but wouldn't it be better to specify in which cases we want to run this? This condition will be executed for is_sle_micro/is_microos (even though sle micro doesn't use main_common).

So, I would probably prefer:
```
loadtest 'console/systemd_wo_udev' if (is_sle('>=15-sp4') || is_leap('>=15.4') || is_tumbleweed);
```

Reply:
I guess this function might be too generic, so better to be explicit. I took a shortcut :)

Reply:
Alright, it should be the same as in `test_systemd_install`.